packages/dds/tree/src/test/memory/tableTree.spec.ts

```diff
@@ -50,7 +50,7 @@ function createUndoBenchmark({
 
 		public async run(): Promise<void> {
 			assert(this.undoRedoManager !== undefined, "undoRedoManager is not initialized");
-			for (let i = 0; i < stackCount; i++) {
+			for (let i = 0; i < operationCount * editsPerOperation; i++) {
```

Comment:
Nit: I would probably save this off as a variable, otherwise the multiplication operation will be repeated with each iteration. Same for the other sites.

Reply:
Updated